cmd/minishift/cmd/util/util.go

```diff
@@ -50,3 +63,30 @@ func ExitIfNotRunning(driver drivers.Driver, machineName string) {
 		atexit.ExitWithMessage(0, fmt.Sprintf("The execution of this command requires a running '%s' VM, but there is currently none.", machineName))
 	}
 }
+
+// UnpackAddons will unpack the default addons into addons default dir
+func UnpackAddons(addonsDir string) error {
+	for _, asset := range DefaultAssets {
+		if err := bindata.RestoreAssets(addonsDir, asset); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CreateUpdateMarker(markerPath string, data UpdateMarker) error {
+	f, err := os.OpenFile(markerPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	defer f.Close()
+	if err != nil {
+		return err
+	}
+
+	b, err := json.Marshal(data)
+	if err != nil {
+		fmt.Println("error:", err)
```

Comment:
That seems wrong. Is this intended? You just log an error and continue. Should you not return the error?